versions/3.4.0/patch

```diff
@@ -34,7 +34,7 @@ index 64efcaa4..d093e2b0 100644
              else:
                  CCM_CLUSTER = CCMCluster(path, cluster_name, **CCM_KWARGS)
 -            CCM_CLUSTER.set_configuration_options({'start_native_transport': True})
-+            CCM_CLUSTER.set_configuration_options({'start_native_transport': True, 'experimental': True})
```

Comment:
I'm afarid it'll just be forgotten here..
I prefer to remove experimental completely, or have it as global parameter somehow for the entire run if it's possible.
So, we can have an entire run with experimental: True and one without it.